products/beats.md

```diff
@@ -33,6 +33,7 @@ releases:
     eol: false  # later of 2027-10-15 or 18 months after the release date of 10.0
     latest: "9.0.1"
     latestReleaseDate: 2025-04-29
+    link: https://www.elastic.co/docs/release-notes/beats#beats-9.0.1-release-notes
```

Comment:
```suggestion
    link: https://www.elastic.co/docs/release-notes/beats#beats-__LATEST__-release-notes
```

Trying this would be nice, and even if it does not work for future versions it won't break the link.

Reply:
to be honest they might change the naming Schema , ( for example they have used 900 instead of 9.0.0  ). So i suggest we merge with current shape and give a look after first release of upcomming (minor )version

Reply:
I'm merging with current shape and will revisit after first new release